statsmodels/tsa/statespace/mlemodel.py

```diff
@@ -159,6 +164,11 @@ def fit(self, start_params=None, transformed=True,
         return_params : boolean, optional
             Whether or not to return only the array of maximizing parameters.
             Default is False.
+        hessian_method : {'opg','oim','cs'}, optional
+            The method by which the Hessian is numerically approximated. 'opg'
+            uses outer product of gradients, 'oim' uses the information
+            matrix formula from Harvey (1989), and 'cs' uses second-order
```

Comment:
i don't see why users should set hessian method. We should have `cov_type` as explicit keyword, following my pattern in other models.